src/current/cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -167,220 +70,109 @@ In this tutorial, you will create a CockroachDB {{ site.data.products.dedicated
-              + sql_dns    = (known after apply)
-              + ui_dns     = (known after apply)
-            },
-        ]
-      + serverless        = {
-          + routing_id  = (known after apply)
-          + spend_limit = 0
-        }
-      + state             = (known after apply)
-    }
+In this tutorial, you will create a CockroachDB {{ site.data.products.basic }} cluster.
 
-  # cockroach_sql_user.example will be created
-  + resource "cockroach_sql_user" "example" {
-      + cluster_id = (known after apply)
-      + id         = (known after apply)
-      + name       = "maxroach"
-      + password   = (sensitive value)
+1. In a terminal create a new file named `main.tf` with the following contents:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ hcl
+    resource "cockroach_cluster" "basic" {
+      name           = "cockroach-basic"
+      cloud_provider = "GCP"
+      plan           = "BASIC"
+      serverless     = {}
+      regions = [
+        {
+          name = "us-east1"
+        }
+      ]
+      delete_protection = false
     }
+    ~~~
+      - Replace `cockroach-basic` with a name for the cluster.
+      - Set `cloud_provider` to `AWS` `AZURE`, or `GCP`.
+      - Under `regions`, add the names of one or more regions for the cluster.
+      - To optionally enable [deletion protection]({% link cockroachcloud/basic-cluster-management.md %}#enable-deletion-protection), set `delete_protection` to `true`.
+{{ remaining_steps }}
 
-Plan: 2 to add, 0 to change, 0 to destroy.
+</section>
 
-Do you want to perform these actions?
-  Terraform will perform the actions described above.
-  Only 'yes' will be accepted to approve.
+<section class="filter-content" markdown="1" data-scope="standard">
 
-  Enter a value: yes
+In this tutorial, you will create a CockroachDB {{ site.data.products.standard }} cluster.
 
-cockroach_cluster.example: Creating...
-cockroach_cluster.example: Creation complete after 5s [id=1aaae1f8-19e2-4653-ba62-db16de2a84b9]
-cockroach_sql_user.example: Creating...
-cockroach_sql_user.example: Creation complete after 2s [id=1aaae1f8-19e2-4653-ba62-db16de2a84b9:maxroach]
+1. In a terminal create a new file named `main.tf` with the following contents:
 
-Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
-~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ hcl
+    resource "cockroach_cluster" "standard" {
+      name           = "cockroach-standard"
+      cloud_provider = "GCP"
+      plan           = "STANDARD"
+      serverless = {
+        usage_limits = {
+          provisioned_vcpus = 2
+        }
+      }
+      regions = [
+        {
+          name = "us-east1"
+        }
+      ]
+      delete_protection = false
+    }
+    ~~~
+      - Replace `cockroach-standard` with a name for the cluster.
+      - Set `cloud_provider` to `AWS` `AZURE`, or `GCP`.
+      - Under `usage_limits`, set `provisioned_vcpus` to the number of vCPUs required per node.
```

Comment:
`provisioned_vcpus` is not per node, because Standard clusters do not have nodes (nodes are a concept that only applies to Advanced clusters). Instead, `provisioned_vcpus` is the total number of vCPUs for the whole cluster.